src/api/comments.ts

```diff
@@ -0,0 +1,28 @@
+const BASE_URL = 'https://mate.academy/students-api/';
+
+export const getComments = async (postId: number) => {
+  try {
+    const comments = await fetch(`${BASE_URL}comments`);
+    const commentsJson: Comment[] = await comments.json();
+
+    return commentsJson.filter((comment: Comment) => comment.postId === postId);
```

Comment:
you can use this ability of API to load needed comments 
![image](https://user-images.githubusercontent.com/64437769/134152740-0c072945-eaaf-46a3-939c-ead1df31ccd3.png)